build.py

```diff
@@ -0,0 +1,58 @@
+import os
+import shutil
+from distutils.core import Distribution
+from pathlib import Path
+from subprocess import call
+
+from setuptools_rust import Binding, RustExtension, build_rust
+
+
+class BuildFailed(Exception):
```

Comment:
This class seems to be unused.

Reply:
it was for testing: https://github.com/crypto-org-chain/chainlibpy/pull/52/files#diff-b70bfb027a8ef3ec4ab07a53a3b8ba83e63334ba3ce5ca6d9d773900ed452e5eR58 -- I can perhaps comment it out too

Reply:
I see, thank you.